app/views/user_measure_mailer/created.en.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Measure Assigned</h1>
+
+<p>
+  Hi <%= @name %>,
+</p>
+
+<p>
+  You have a new measure assigned to you <%= @title %>.
```

Comment:
This should also include a link to the measure which is
https://plastic-advocacy.wwf.no/action/[id]

Should we store the client URL and measure path as a global config variable(s)?

Reply:
Regardless, will provide you with the final text shortly - also can easily update myself.

Reply:
@tmfrnz yeah, I think storing it in config is a good idea here 😄

Reply:
@tmfrnz I've updated the templates, let me know what you think